lib/lotus/entity.rb

```diff
@@ -235,6 +252,46 @@ def update(attributes={})
       end
     end
 
+    # Getter for hash of changed attributes.
+    # Return empty hash, if there is no changes
+    #
+    # @since 0.3.0
+    #
+    # @example
+    #   @changed_attributes # => {:title => 'New title'}
+    #
+    def changed_attributes
+      @changed_attributes ||= Hash.new
+    end
+
+    # Return boolean of dirty state for model.
+    # If model changed, return true, else false.
```

Comment:
what do you think about following our conventional `@return [TrueClass, FalseClass] the changed changed indicator